website/site/content/docs/collection-types.md

```diff
@@ -30,6 +30,8 @@ Example:
     - {label: "Body", name: "body", widget: "markdown"}
 ```
 
+**Note:** Folder Collections currently require a `title` field for creating new entries.
```

Comment:
I'd say put this right before "Example:", and add a bit more so it's clear what we mean by "title field":

> Folder collections must have at least one field with the name "title" for creating new entry slugs. That field should use the default "string" widget. The "label" for the field can be any string value.